README.md

```diff
@@ -141,6 +140,9 @@ This project uses the [pyrex](https://github.com/garmin/pyrex) tool to transpare
 
         Boot your NI Linux Real-Time compatible hardware from the recovery media and follow on-screen instructions to perform a factory reset.
 
+    **NOTE** OpenSSH 8.8 and later will by default refuse SHA1 signatures (referred to as `ssh-rsa`); if using SSH (`ssh`, `scp`, `sftp`, etc.) to copy images to a target, it may be necessary to configure the client to use newer authentication or to configure the target to allow the deprecated strategy. See [their release notes](http://openssh.com/txt/release-8.8) for more information, including how to re-enable it.  
+    System Configuration utilities version 22.5 or later should use the newer strategies by default.
+
```

Comment:
I'm not sure we need to include a note into the README about this deprecation. We're not expecting it to impact a substantial portion of users. And those who are impacted have legacy custom SSH scripting. I doubt those users will be looking through the README for information as to why their scripting stopped working.

Reply:
Makes sense to me; it'll be in the changelog anyway, and I only added it since it's mentioned in the work item. I'll drop that commit.